src/fstapp_mnesia_handler.erl

```diff
@@ -18,11 +18,11 @@
 handle_init() ->
 	_ =  mnesia:create_schema([node()]),
 	_ = mnesia:create_table(metrics,[{attributes, record_info(fields, metrics)}]),
-	ok.
+	{ok, ok}.
```

Comment:
it is fine but ugly. It is more usual see things like `{ok, nostate}` or {ok, #{}}`